test/e2e/networking.go

```diff
@@ -150,7 +184,7 @@ var _ = Describe("Networking", func() {
 		By("Waiting for connectivity to be verified")
 		const maxAttempts = 60
 		passed := false
-		var body []byte
+		var body []byte //once response OK, we evaluate this variable for test pass/fail.
```

Comment:
Move to the previous line. I don't like having both comment and code in the same line.